[2025-04-09T16:34:41.852+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-04-09T16:34:41.872+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: extract_raw_data_pipeline.run_dbt_trusted manual__2025-04-09T16:34:15.402601+00:00 [queued]>
[2025-04-09T16:34:41.881+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: extract_raw_data_pipeline.run_dbt_trusted manual__2025-04-09T16:34:15.402601+00:00 [queued]>
[2025-04-09T16:34:41.881+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2025-04-09T16:34:42.011+0000] {taskinstance.py:2888} INFO - Executing <Task(BashOperator): run_dbt_trusted> on 2025-04-09 16:34:15.402601+00:00
[2025-04-09T16:34:42.017+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=1343) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-04-09T16:34:42.019+0000] {standard_task_runner.py:72} INFO - Started process 1345 to run task
[2025-04-09T16:34:42.019+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'extract_raw_data_pipeline', 'run_dbt_trusted', 'manual__2025-04-09T16:34:15.402601+00:00', '--job-id', '98', '--raw', '--subdir', 'DAGS_FOLDER/driven_data_pipeline.py', '--cfg-path', '/tmp/tmpm2kem7ad']
[2025-04-09T16:34:42.020+0000] {standard_task_runner.py:105} INFO - Job 98: Subtask run_dbt_trusted
[2025-04-09T16:34:42.115+0000] {task_command.py:467} INFO - Running <TaskInstance: extract_raw_data_pipeline.run_dbt_trusted manual__2025-04-09T16:34:15.402601+00:00 [running]> on host 5a8aeda502d0
[2025-04-09T16:34:42.254+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='extract_raw_data_pipeline' AIRFLOW_CTX_TASK_ID='run_dbt_trusted' AIRFLOW_CTX_EXECUTION_DATE='2025-04-09T16:34:15.402601+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-09T16:34:15.402601+00:00'
[2025-04-09T16:34:42.256+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-04-09T16:34:42.276+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-04-09T16:34:42.280+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'set -x; cd /opt/***/dbt && dbt run --select tag:trusted']
[2025-04-09T16:34:42.296+0000] {subprocess.py:86} INFO - Output:
[2025-04-09T16:34:42.298+0000] {subprocess.py:93} INFO - + cd /opt/***/dbt
[2025-04-09T16:34:42.299+0000] {subprocess.py:93} INFO - + dbt run --select tag:trusted
[2025-04-09T16:34:44.591+0000] {subprocess.py:93} INFO - [0m16:34:44  Running with dbt=1.8.0
[2025-04-09T16:34:44.893+0000] {subprocess.py:93} INFO - [0m16:34:44  Registered adapter: postgres=1.8.2
[2025-04-09T16:34:45.473+0000] {subprocess.py:93} INFO - [0m16:34:45  Found 9 models, 10 sources, 428 macros
[2025-04-09T16:34:45.477+0000] {subprocess.py:93} INFO - [0m16:34:45
[2025-04-09T16:34:45.661+0000] {subprocess.py:93} INFO - [0m16:34:45  Concurrency: 1 threads (target='dev')
[2025-04-09T16:34:45.662+0000] {subprocess.py:93} INFO - [0m16:34:45
[2025-04-09T16:34:45.683+0000] {subprocess.py:93} INFO - [0m16:34:45  1 of 4 START sql table model driven_trusted.non_pii_data ....................... [RUN]
[2025-04-09T16:34:51.237+0000] {subprocess.py:93} INFO - [0m16:34:51  1 of 4 OK created sql table model driven_trusted.non_pii_data .................. [[32mSELECT 152294[0m in 5.55s]
[2025-04-09T16:34:51.240+0000] {subprocess.py:93} INFO - [0m16:34:51  2 of 4 START sql table model driven_trusted.payment_data ....................... [RUN]
[2025-04-09T16:34:53.335+0000] {subprocess.py:93} INFO - [0m16:34:53  2 of 4 OK created sql table model driven_trusted.payment_data .................. [[32mSELECT 152294[0m in 2.09s]
[2025-04-09T16:34:53.339+0000] {subprocess.py:93} INFO - [0m16:34:53  3 of 4 START sql table model driven_trusted.pii_data ........................... [RUN]
[2025-04-09T16:34:58.690+0000] {subprocess.py:93} INFO - [0m16:34:58  3 of 4 OK created sql table model driven_trusted.pii_data ...................... [[32mSELECT 152294[0m in 5.35s]
[2025-04-09T16:34:58.693+0000] {subprocess.py:93} INFO - [0m16:34:58  4 of 4 START sql table model driven_trusted.technical_data ..................... [RUN]
[2025-04-09T16:35:00.594+0000] {subprocess.py:93} INFO - [0m16:35:00  4 of 4 OK created sql table model driven_trusted.technical_data ................ [[32mSELECT 152294[0m in 1.90s]
[2025-04-09T16:35:00.614+0000] {subprocess.py:93} INFO - [0m16:35:00
[2025-04-09T16:35:00.614+0000] {subprocess.py:93} INFO - [0m16:35:00  Finished running 4 table models in 0 hours 0 minutes and 15.14 seconds (15.14s).
[2025-04-09T16:35:00.692+0000] {subprocess.py:93} INFO - [0m16:35:00
[2025-04-09T16:35:00.693+0000] {subprocess.py:93} INFO - [0m16:35:00  [32mCompleted successfully[0m
[2025-04-09T16:35:00.694+0000] {subprocess.py:93} INFO - [0m16:35:00
[2025-04-09T16:35:00.695+0000] {subprocess.py:93} INFO - [0m16:35:00  Done. PASS=4 WARN=0 ERROR=0 SKIP=0 TOTAL=4
[2025-04-09T16:35:01.725+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-04-09T16:35:01.789+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-04-09T16:35:01.790+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=extract_raw_data_pipeline, task_id=run_dbt_trusted, run_id=manual__2025-04-09T16:34:15.402601+00:00, execution_date=20250409T163415, start_date=20250409T163441, end_date=20250409T163501
[2025-04-09T16:35:01.856+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-04-09T16:35:01.876+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-04-09T16:35:01.910+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
